### PR TITLE
Fix Scala 2.12 build

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,15 @@
 sudo: required
 
+scala:
+  - 2.12.10
+  - 2.13.1
```
Comment (sksamuel) [inline .travis.yml]: I'm not sure this works, you need to do the sbt ++ trick.
See https://github.com/sksamuel/avro4s/blob/master/.travis.yml

Comment (gmethvin) [reply .travis.yml]: oh I knew I forgot something. Thanks!

**Comment by gmethvin** [comment]
OK now we're good for a release @sksamuel.

**Comment by sksamuel** [comment]
2.4.5 is released

**Pull request merged**